dataland-frontend/tests/e2e/fixtures/custom_mocks/StoredDataRequestsFaker.ts

```diff
@@ -60,16 +60,30 @@ export function generateStoredDataRequest(): StoredDataRequest {
   const minimalNumberOfMessageObjects = 1;
   const messageHistory = generateArray(() => generateStoredDataRequestMessage(), minimalNumberOfMessageObjects);
   const timeOffsetBetweenCreationAndLastModified = 500;
+  const creationTime = generateInt(timeOffsetBetweenCreationAndLastModified);
+  const lastModifiedTime = generateInt(500) + timeOffsetBetweenCreationAndLastModified;
```

Comment:
magic number, atleast use a constant at the top of this file

Reply:
As suggested, a constant is now used at the top of the file